CONTRIBUTING.md

```diff
@@ -149,6 +149,16 @@ git push
 
 **IMPORTANT**: By submitting a patch, you agree to the [License](#license).
 
+### Privacy Notice
+
+We would like to make you aware that contributing to OpenTTD via git will permanently store the name and email address you provide as well as the actual commit and the time and date you made it inside git's version history.
```

Comment:
Rather than "the actual commit", perhaps "the actual change"?

Maybe also a sentence about commit histories longer than one, are they preserved when a change is merged, and whose responsibility is it to do any squashing?